pkgs/top-level/all-packages.nix

```diff
@@ -10223,6 +10223,8 @@ in
 
   riak = callPackage ../servers/nosql/riak/2.1.1.nix { };
 
+  stanchion = callPackage ../servers/nosql/riak-cs/stanchion.nix { };
```

Comment:
I think it's better to override the erlang interpreter in the callPackage scope and just use the erlang attribute in the derivation.

```
  stanchion = callPackage ../servers/nosql/riak-cs/stanchion.nix {
    erlang = erlang_basho_R16B03;
  };
```